distracting_control/gym_env.py

```diff
@@ -14,7 +14,10 @@ def convert_dm_control_to_gym_space(dm_control_space, **kwargs):
                            dtype=dm_control_space.dtype)
         assert space.shape == dm_control_space.shape
         return space
-    elif isinstance(dm_control_space, specs.Array) and not isinstance(dm_control_space, specs.BoundedArray):
```

Comment:
@sebimarkgraf Can you add this `not isinstance(dm_control_space, specs.BoundedArray)` part in the edited version, just for completion?

Reply:
I will be away for a bunch of meetings today. Just want to point you to the new committee I made above. There are now two commits.

Thanks takuma! 

The test runs by 

```bash

make test

```

And all tests need to past

Reply:
@takuma-yoneda Added again

Reply:
@geyang Tests do not pass for me on main. 
Seems to be due to missing dependencies.

I am not sure if it is a good idea to fix the tests and create the spec changes in one PR.